packages/monosize/src/commands/measure.mts

```diff
@@ -12,15 +12,16 @@ import { readConfig } from '../utils/readConfig.mjs';
 import type { CliOptions } from '../index.mjs';
 import type { BuildResult } from '../types.mjs';
 
-type MeasureOptions = CliOptions & { debug: boolean };
+export type MeasureOptions = CliOptions & { debug: boolean };
 
 async function measure(options: MeasureOptions) {
   const { debug = false, quiet } = options;
 
   const startTime = process.hrtime();
-  const artifactsDir = path.resolve(process.cwd(), 'dist', 'monosize');
+  const artifactsDir = path.resolve(process.cwd(), 'dist', 'bundle-size');
 
   await fs.promises.rm(artifactsDir, { recursive: true, force: true });
+  await fs.promises.mkdir(artifactsDir, { recursive: true });
```

Comment:
Usually it's created by a bundler, but we need to create a directory to ensure that it exists